README.md

```diff
@@ -1,4 +1,4 @@
-[![GSoC Logo](https://developers.google.com/open-source/gsoc/resources/downloads/GSoC-logo-horizontal.svg)](https://summerofcode.withgoogle.com/projects/#5747767629578240)
+[![GSoC Logo](/title_img/spdx-online-tool.jpg)](https://summerofcode.withgoogle.com/projects/#5747767629578240)
```

Comment:
It should be either `./title_img/spdx-online-tool.jpg` or link to image `https://github.com/spdx/spdx-online-tools/tree/master/title_img/spdx-online-tool.jpg`

Reply:
thanks, @rtgdk for helping, although it was working this way as well!

Reply:
It wasn't showing in Pycharm. Also, relative path will work well with other tools.